### ISSUE TITLE
Changes since the last version

### ISSUE THREAD

**Issue by ericprud** [shexSpec/shex#118]
@gkellogg asked for this and it seems especially important given that a proposed definition for extends [adds a parameter to satisfies](https://shex.io/extends-new-param/#satisfies). @gkellogg , have you got a good tool? Diffs like [this](https://pr-preview.s3.amazonaws.com/shexSpec/spec/40/ed1f660...67d10c1.html) tend to mangle &lt;pre/&gt;s. Also, I don't know how to point it at e.g. the [2.1 spec](http://shex.io/shex-semantics-20191008/) and the  [extends-new-param](https://github.com/shexSpec/spec/compare/extends-new-param?expand=1) branch. (The first step is probably to finalize 2.1 and give it a maintenance branch like [2.0](https://github.com/shexSpec/spec/tree/ShEx2.0).)

Barring a good enough tool, we can do the job a couple times by hand as we finalize 2.2.

**Comment by gkellogg** [comment]
IMO, diffs are actually less useful than an appendix summarizing the change with a reference to the associated PR. PR Preciew does a good job of a diff for that specific change. I think it still uses Shane’s original htmldiff tool.

**Comment by ericprud** [comment]
Makes sense. I think the appendix can say "there's an extra parameter at the end or `satisfies`" but it might be nice to see that highlighted in the text as well.

**Comment by gkellogg** [comment]
I think The ReSpec team is working on a way to automate this for spec updates. It’s complicated by the new process document which didn’t consider the need of tooling or the burden on editors sufficiently. But, it seems like a good workflow to just list the PRs from the original spec to the updated version and let the tools just do the right thing.
